### PR TITLE
Filled Google Colab page 

### PR DESCRIPTION
**Pull request by casruger**
Filled the Google Colab page with text + added an image used in that page.

### CODE REVIEW

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 1]
```diff
@@ -1,29 +1,67 @@
...
+date: 2022-04-08T10:01:14+05:30
+draft: false
+author: "Cas Ruger"
+# authorlink: "https:/"
+aliases:
+  - /use/colab
+  
+---
+
+## What is Google Colab
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:14]: Use a question mark - What is Google Colab?

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 2]
```diff
@@ -1,29 +1,67 @@
...
+Colab is a product from Google Research that allows anybody with an internet connection to write and execute python code through a browser. Colab is free to use, and there is no setup on your computer needed. It is especially useful if you have a slow computer since Google hosts your Jupyter notebook and thus uses their GPU free of charge. Or if you want to work together on the same code, since you can collaborate on the same cloud file with multiple editors. 
+
+## How to get started
+It is time to create your own Colab notebook.
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:18]: End with an exclamation mark! "Active voice"

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 3]
```diff
@@ -1,29 +1,67 @@
...
+
+
+1. Go to https://colab.research.google.com/ 
+2. Press **New notebook**, or if you already have a file, choose this one from the list
+3. Save your notebook to either Google drive or GitHub via **File** > **Save a copy in ...**
+
+
+## How to use Colab
+Once you are in your Colab notebook you can use it just like any offline Jupiter notebook.
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:27]: Run a quick check on typos - I think it's Jupyter Notebook, not Jupiter Notebook.

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 4]
```diff
@@ -1,29 +1,67 @@
...
+1. Go to https://colab.research.google.com/ 
+2. Press **New notebook**, or if you already have a file, choose this one from the list
+3. Save your notebook to either Google drive or GitHub via **File** > **Save a copy in ...**
+
+
+## How to use Colab
+Once you are in your Colab notebook you can use it just like any offline Jupiter notebook. 
+
+Type your code in the darker gray box en pres the run arrow to run the code. Your output will be placed below the box. To create a new code box, press **+ Code** in the navigation bar. 
+
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:30]: Can you please add a screenshot of google colab below?

Comment (hannesdatta) [reply content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:30]: Happy to show you btw how to do that.

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 5]
```diff
@@ -1,29 +1,67 @@
...
+
+### Import packages
+Colab environment already has a number of pre-installed packages such as pandas, pytorch, scipy, tenserflow and numpy. To check all installed packages use **pip freeze** 
+```bash 
+!pip freeze.
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:35]: Is it correct to end this command with a DOT? Same below?

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 6]
```diff
@@ -1,29 +1,67 @@
...
+
+
+## How to use Colab
+Once you are in your Colab notebook you can use it just like any offline Jupiter notebook. 
+
+Type your code in the darker gray box en pres the run arrow to run the code. Your output will be placed below the box. To create a new code box, press **+ Code** in the navigation bar. 
+
+
+### Import packages
+Colab environment already has a number of pre-installed packages such as pandas, pytorch, scipy, tenserflow and numpy. To check all installed packages use **pip freeze**
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:33]: Wrap package names in `` so it is rendered as code that can be copy pasted, e.g.,

`pandas`, `pytorch`, `scipy`, etc.

File: content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md [hunk 7]
```diff
@@ -1,29 +1,67 @@
...
+```
+
+If a package is not yet installed, install the package inside the Colab notebook by adding the code **!pip install {package name}**
+```bash 
+!pip install cartopy. 
+import cartopy
+```
+
+### Connect to google drive
+The easiest way to upload files is to connect your Colab file to your Google drive.
```
Comment (hannesdatta) [inline content/building-blocks/configure-your-computer/infrastructure-choice/google-colab.md:50]: Explain why uploading is necessary.

**Review by hannesdatta — CHANGES_REQUESTED** [review]
Good job, Cas! I have provided a few comments. Why don't you take 10 minutes to implement them and then we can put the block live! Looking forward!!!

**Comment by casruger** [comment]
Included the feedback of Hannes in the file and added a screenshot of Google Colab

**Pull request merged**